src/cyminmax.pyx

```diff
@@ -23,7 +23,8 @@ def minmax(a):
         out(numpy.ndarray):        an array with the min and max values.
     """
 
-    arr = numpy.asanyarray(a)
+    cpython.Py_INCREF(a)
+    arr = numpy.PyArray_EnsureAnyArray(a)
```

Comment:
Need to increase the reference count on `a` as `PyArray_EnsureArray` (called by `PyArray_EnsureAnyArray`) steals a reference to `a`. If the reference count is not increased on `a`, this results in a segmentation fault (error number `11`).